File: go/ciclofor.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var A, C int

	// Pedir el primer valor
	fmt.Print("Ingrese el primer valor: ")
	fmt.Scan(&A)

	// Pedir el segundo valor
	fmt.Print("Ingrese el segundo valor: ")
	fmt.Scan(&C)

	// Calcular la potencia
	valor := math.Pow(float64(A), float64(C))

	// Imprimir el resultado
	fmt.Printf("La potencia de %d sobre %d es: %.0f\n", A, C, valor)
}

File: JavaScript/ciclofor.js
// Pedir los valores al usuario//
//se usa prompt() para pedir al usuario que ingrese los valores A y C//
const A = parseInt(prompt("Ingrese el primer valor:"));
const C = parseInt(prompt("Ingrese el segundo valor:"));

// Calcular la potencia//
//Math.pow(A, C) calcula la potencia de A elevado a C//
const valor = Math.pow(A, C);

// Mostrar el resultado//
//alert para mostrar el resultad0//
alert(`La potencia de ${A} sobre ${C} es: ${valor}`);
